web/pageComponents/shared/portableText/IngressText.tsx

```diff
@@ -9,7 +9,7 @@ import { Text, List } from '@components'
 const { Item } = List
 
 const StyledList = styled(List)`
-  font-size: var(--typeScale-2);
```

Comment:
does it make sense to remove it for all use cases? maybe it was intentional for ingress?